core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.hpp

```diff
@@ -813,8 +813,8 @@ class TeamPolicyInternal<Kokkos::Experimental::OpenMPTarget, Properties...>
   int m_vector_length;
   int m_team_alloc;
   int m_team_iter;
-  size_t m_team_scratch_size[2];
-  size_t m_thread_scratch_size[2];
+  std::array<size_t, 2> m_team_scratch_size;
```

Comment:
please don't I am not confident about std::array in all the OpenMPTarget compilers we try to use.

Reply:
nvm in OpenMPTarget we don't map the policy.